pkg/kube/secret/secret.go

```diff
@@ -13,6 +13,7 @@ import (
 )
 
 type Getter interface {
+	IsNotFound(err error) bool
```

Comment:
I decided to add this as method for the interface. This way community will just wrap it with `apiErrors.IsNotFound()` while enterprise will add additional logic based on message returned by the vault secret

Reply:
ah, I see -- I already added it to master. do you think we still need this?